### ISSUE TITLE
Investigate Voice Control compatibility

### ISSUE THREAD

**Issue by enriquesanchez** [WordPress/gutenberg#18786]
**Describe the bug**
Apple's Voice Control is a new accessibility feature [included in macOS Catalina](https://support.apple.com/guide/mac-help/control-your-mac-and-apps-using-voice-control-mh40719/mac).
[ Here's a video that shows how it works](https://www.youtube.com/watch?v=aqoXFCCTfm4).

It's pretty cool and powerful and makes me very excited, which is why I've been learning it and trying to get used to it. 

It's also fair to say it's a rather new feature, so there's still a lot of unknowns and to my knowledge, little documentation and general information from third parties. Having said that, I've performed some basic tests with WP and Gutenberg and found that it *almost* works.

The only issue I've found so far (again, I'm starting to learn the tool), is with Gutenberg's popovers. I've tested it with both the the block toolbar and inspector. In both cases, once a popover is opened, Voice Control cannot interact with the web page anymore and I have to resort to restarting Voice Control.

- [Test 01 video](https://cldup.com/h0sNreuRri.mov)
- [Test 02 video](https://cldup.com/ItOD9Du7rg.mov)

It'd be great if we can figure out what's happening and see if it's something we can solve.

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.1
 - Browser: Safari
 - Version: 13.0.3

**Comment by tellthemachines** [comment]
I ran a quick check with Voice Control in Safari and was able to summon up the numbers menu after opening the block popup as per your first video. I did find Voice Control in general somewhat unresponsive though - it doesn't always recognise my commands (not sure if it's an accent issue or something). 

Are you able to interact with popovers on other apps/websites?

**Comment by enriquesanchez** [comment]
@tellthemachines Thank for testing!

I ran another test and was able to interact with the popovers for a few tries, it then again started failing.

I tried your suggestion of testing on other sites. I tried it with Github's dropdown menus and I ran into the same issue.

This may very well be an issue with Voice Control in general, or my setup 🤔
